theano/sandbox/tests/test_rng_mrg.py

```diff
@@ -1189,6 +1190,71 @@ def test_validate_input_types_gpuarray_backend():
         rstate = gpuarray_shared_constructor(rstate)
```

Comment:
I think it would be better to import `theano.gpuarray.tests.config` in here, or just `raise SkipTest('pygpu not installed')` here, so the test does not get executed when it is not necessary.